src/api.raml

```diff
@@ -442,6 +458,27 @@ traits:
                                     displayName: '15278'
                                     value: !include samples/article-vor/v2/15278.json
                         application/vnd.elife.article-vor+json;version=3:
+                            schema: !include model/article-vor.v3.json
+                            examples:
+                                minimum:
+                                    displayName: Minimum
+                                    value: !include samples/article-vor/v3/minimum.json
+                                complete:
+                                    displayName: Complete
+                                    value: !include samples/article-vor/v3/complete.json
+                                unpublishedMinimum:
+                                    displayName: PreviewMinimum
+                                    value: !include samples/article-vor/v3/unpublished-minimum.json
+                                unpublishedComplete:
+                                    displayName: PreviewComplete
+                                    value: !include samples/article-vor/v3/unpublished-complete.json
+                                00569:
+                                    displayName: '00569'
+                                    value: !include samples/article-vor/v3/00569.json
+                                15278:
+                                    displayName: '15278'
+                                    value: !include samples/article-vor/v3/15278.json
+                        application/vnd.elife.article-vor+json;version=4:
                             (experimental):
                             schema: !include model/article-vor.v3.json
                             examples:
```

Comment:
do these examples need to be updated from

    !include samples/article-vor/v3/minimum.json

to

    !include samples/article-vor/v4/minimum.json

? It looks like you have the files included in the PR